packages/studio/src/components/ActiveComponentPropEditors.tsx

```diff
@@ -32,7 +32,7 @@ export default function ActiveComponentPropEditors(props: {
 
   const propValues = activeComponentState.props;
   return (
-    <div className="overflow-x-auto">
+    <div className="overflow-x-auto grow flex flex-col h-fill">
```

Comment:
what does `h-fill` do? I'm only familiar with `h-full`

Reply:
Meant to put `h-full` but it seems to not be necessary here anyway.